cmasher/utils.py

```diff
@@ -233,6 +239,115 @@ def _get_cmap_perceptual_rank(
 
 
 # %% FUNCTIONS
+# This function combines multiple colormaps at given nodes
+def combine_cmaps(
+    cmaps: list[Union[Colormap, str]],
+    *,
+    nodes: Optional[list[float]] = None,
+    n_rgb_levels: int = 256,
+    combined_cmap_name: str = "combined_cmap",
+) -> LinearSegmentedColormap:
+    """Create a composite matplotlib colormap by combining multiple colormaps.
+
+    Parameters
+    ----------
+        cmaps: list of Colormap or colormap names (str)
+            list of colormaps to combine.
+        nodes: list or numpy array of floats. Defaults: equal divisions.
+            The blending points between colormaps, in the range [0, 1].
+        n_rgb_levels: int. Defaults: 256.
+            Number of RGB levels for each colormap segment.
+        combined_cmap_name: str. Defaults: "combined_cmap".
+            name of the combined Colormap.
+
+    Returns
+    -------
+        Colormap: The composite colormap.
+
+    Raises
+    ------
+        TypeError: If the list contains mixed datatypes or invalid
+            colormap names.
+        ValueError: If the cmaps contain only one single colormap,
+            or if the number of nodes is not one less than the number
+            of colormaps, or if the nodes do not contain incrementing values
+            between 0.0 and 1.0.
+
+    Note
+    ----
+        The colormaps are combined from bottom to top.
```

Comment:
I'm not sure this is helpful: colormaps don't have an absolute orientation, and may be displayed horizontally. Is there a way to rephrase this note that would work independently of orientation ?

Reply:
I was thinking the same about the necessity of having the note and the phrasing of the note yesterday. 

As the `LinearSegmentedColormap.from_list()` function would combine the cmaps from **lower value end to higher value end** (if not reversed, that is bottom to top). So this is expected behaviour for the cmaps to start from the lower value end. But meanwhile for me, I expected the cmaps to be combined from top to bottom at first attempt, when I don't have much understanding of `Colormap`.

Maybe we could remove this note altogether? As the behaviour is expected?

Or rephrase it to something like "The colormaps are combined from low value to high value end."? What would your suggestion be?